bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -4513,4 +4513,14 @@ spec:
       name: externalhealthmonitorcontroller
     - image: quay.io/dell/container-storage-modules/csi-metadata-retriever:v1.10.0
       name: metadataretriever
+    - image: docker.io/nginxinc/nginx-unprivileged:1.27
+      name: nginx
+    - image: docker.io/redis:7.4.1-alpine
+      name: redis
+    - image: docker.io/rediscommander/redis-commander:latest
+      name: redis-commander
+    - image: docker.io/openpolicyagent/opa:latest
```

Comment:
We're using opa:0.70.0 in the samples: https://github.com/dell/csm-operator/blob/75f0aa4059d2a8696f4234562c207e4d400bbb15/samples/authorization/csm_authorization_proxy_server_v220.yaml#L41.

Is that in this csv too?

Reply:
The CSV had latest. I updated the CSV to use the 0.70.0 version since that's what we tested.